src/bindings/python/src/openvino/frontend/onnx/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Package: openvino
+Low level wrappers for the FrontEnd c++ api.
+"""
+
+# flake8: noqa
+
+from openvino.utils import add_openvino_libs_to_path
+
+add_openvino_libs_to_path()
```

Comment:
to be honest, I'm not sure. @akuporos could you please comment?

Reply:
I think this is required because we don't know which particular module will be imported first. To make sure that those paths are always updated correctly, we need to have this call in all `__init__.py` files. @jiwaszki do you know anything about this?

Reply:
@ilyachur 
This piece of code is needed in all __init__.py files due-to the problem on `windows` and `python3.8+`.
Starting from `python3.8` the way how python finds libs on `windows` has changed. 

CC @mryzhov

Reply:
@ilyachur can we resolve a conversation?